shard.yml

```diff
@@ -12,7 +12,8 @@ dependencies:
     github: kemalcr/spec-kemal
     branch: master
   scrypt:
-    github: ysbaddaden/scrypt-crystal
+    github: dscottboggs/scrypt-crystal
```

Comment:
Have you done a PR for this?

Reply:
https://github.com/ysbaddaden/scrypt-crystal/pull/6